help/view-and-browse-images.md

```diff
@@ -1,36 +1,67 @@
-# View and browse images
+# View images and videos
 
-When someone pastes or attaches an image to a message, Zulip shows a small
-preview. Click on the image preview to open the **image viewer**.
+When someone pastes or attaches an image or video to a message, Zulip shows a
+small preview. Similarly, links to images on the web or YouTube videos will
+also generate a small preview. You can open the image viewer to see an image
+in more detail, download it, or watch a video.
 
```

Comment:
I think the intro needs to be adjusted in light of https://zulip.com/help/allow-image-link-previews (previews can be turned off).

Could you please check whether video previews are controlled by the same setting? https://zulip.com/help/allow-image-link-previews should probably be updated as well.

Reply:
Oh, and we actually need to tweak the instructions for opening the preview to something like this, since there might not be a preview:

> Click an image name or preview to open the **image viewer**.

Reply:
> Could you please check whether video previews are controlled by the same setting? https://zulip.com/help/allow-image-link-previews should probably be updated as well.

I checked and it turns out that only YouTube videos are controlled by the same setting. Video previous are always shown for uploaded videos regardless of whether the setting is enabled or disabled.

Reply:
> Oh, and we actually need to tweak the instructions for opening the preview to something like this, since there might not be a preview:
> 
> > Click an image name or preview to open the **image viewer**.

I double checked, and it looks like the user has to click on an image preview to open the image viewer. Otherwise, the image will open up in a new tab.

Reply:
Huh, I'm confused now why I thought that worked.

So maybe there's no way to get to the image viewer if previews are turned off? I tried `V`, but that didn't seem to work either.

Reply:
> I checked and it turns out that only YouTube videos are controlled by the same setting. Video previews are always shown for uploaded videos regardless of whether the setting is enabled or disabled.

Oh, this sounds like a bug. I'll flag it in #issues and mention you. But we should just document it as if the setting controls all image and video previews, as intended.

> Similarly, links to images on the web or YouTube videos will also generate a small preview [if this setting is turned on](https://zulip.com/help/allow-image-link-previews)...

I have a vague recollection that language we use elsewhere is more like:

> Similarly, links to images and videos will also generate a small preview, unless previews are [disabled](https://zulip.com/help/allow-image-link-previews) in your organization.

Reply:
Ok, I also confirmed with @timabbott that there is intentionally no lightbox if image previews are turned off. So let's just make sure the documentation is written to be consistent with that, without being too distracting for the more common situation where image previews are enabled.